packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -39,6 +39,33 @@ import 'theme_data.dart';
 /// [secondary] widget is placed on the opposite side. This maps to the
 /// [ListTile.leading] and [ListTile.trailing] properties of [ListTile].
 ///
+/// This widget requires a [Material] widget ancestor in the tree to paint
+/// itself on, which is typically provided by the app's [Scaffold].
+/// The [tileColor], and [selectedTileColor] are not painted by the
+/// [CheckboxListTile] itself but by the [Material] widget ancestor.
+/// In this case, one can wrap a [Material] widget around the [CheckboxListTile],
+/// e.g.:
+///
```

Comment:
I missed this last time, apologies. This should be wrapped in a `snippet` so it is subject to analysis. That way of we change the code, this will get updated.

```suggestion
///
/// {@tool snippet}
```